.cargo/config.toml

```diff
@@ -6,6 +6,8 @@ rustflags = [
     # See: https://github.com/rust-lang/cargo/issues/5034
     #      https://github.com/EmbarkStudios/rust-ecosystem/issues/22#issuecomment-947011395
 
+    "--cfg=web_sys_unstable_apis",
```

Comment:
1. For sanity, I suggest this flag to take effect just for Web builds, like it were on df11cd6a04369a839249df6dc3c86e88d29ccb99:
```toml
[target.wasm32-unknown-unknown]
rustflags=["--cfg=web_sys_unstable_apis"]
```
2. I wonder if this addition means that `$RUSTFLAGS` can be removed from here:
https://github.com/ruffle-rs/ruffle/blob/1065662e845651bba173236b4a76b74d2881ce56/web/packages/core/package.json#L22

Reply:
For number 2, it does not. These flags are overridden in `package.json`, and need to be until https://github.com/rust-lang/cargo/issues/10271 lands.

Reply:
See https://github.com/ruffle-rs/ruffle/pull/5834

Reply:
I don't know how to apply that condition for just one flag while keeping the others for all targets. Can you tell me how?

Reply:
Doesn't seem possible without duplicating the other flags: https://github.com/rust-lang/cargo/issues/5376

Reply:
Ok, I'll leave it as-is then

Reply:
Since https://github.com/rust-lang/cargo/pull/12148 exists now, perhaps you can add a TODO that once it stabilizes, the clippy lints can be moved to Cargo.toml and this, as the one remaining rustflag, can be specified as `[target.wasm32-unknown-unknown]`

Reply:
I just checked and actually Clippy seems to try to compile our web code with all targets enabled, not just wasm. So we need the flag enabled for all targets unless we can tell Clippy not to do that. I've added the note about that new PR though.